contrib/local.mk.example

```diff
@@ -9,7 +9,7 @@
 
 # By default, the jemalloc family of memory allocation functions are used.
 # Uncomment the following to instead use libc memory allocation functions:
-# CMAKE_EXTRA_FLAGS="-DUSE_JEMALLOC=OFF"
+# DEPS_CMAKE_FLAGS := -DUSE_BUNDLED_JEMALLOC=OFF
```

Comment:
As long as the required logic wouldn't be too complicated then I don't really mind. Then again, this doesn't seem to be much of an issue in (in theory), as if you don't want jemalloc you probably don't have it installed (given how relatively few programs use it). Up to you I suppose. Anyways, I pushed some new commits @fwalch.